dash/frontend/src/app/modules/private/pages/reports/running-vulnerabilities/running-vulnerabilities.component.html

```diff
@@ -1,166 +1,196 @@
+                      [matTooltip]="vulnerability.scanResults">
+              <app-readonly-checkbox
+                [checked]="vulnerability.scanResults === 'Compliant'"></app-readonly-checkbox>
+            </mat-cell>
+          </ng-container>
+          <ng-container matColumnDef="lastScanned" class="column-word-break">
+            <mat-header-cell class="min-width-xs-4" *matHeaderCellDef>Date Scanned</mat-header-cell>
+            <mat-cell class="min-width-4"
+                      *matCellDef="let vulnerability">{{ vulnerability.lastScanned | date: 'M/d/yyyy' }}
+            </mat-cell>
+          </ng-container>
+          <ng-container matColumnDef="totalCritical" class="column-word-break">
+            <mat-header-cell class="min-width-xs-3"
+                             *matHeaderCellDef>{{ VulnerabilitySeverityAbbreviations.CRITICAL |titlecase }}
+            </mat-header-cell>
+            <mat-cell class="min-width-3" *matCellDef="let vulnerability">{{ vulnerability.totalCritical }}
+            </mat-cell>
+          </ng-container>
+          <ng-container matColumnDef="totalMajor">
+            <mat-header-cell class="min-width-xs-3"
+                             *matHeaderCellDef>{{ VulnerabilitySeverityAbbreviations.MAJOR |titlecase }}
+            </mat-header-cell>
+            <mat-cell class="min-width-3" *matCellDef="let vulnerability">{{ vulnerability.totalMajor }}
+            </mat-cell>
+          </ng-container>
+          <ng-container matColumnDef="totalMedium">
+            <mat-header-cell class="min-width-xs-3"
+                             *matHeaderCellDef>{{ VulnerabilitySeverityAbbreviations.MEDIUM |titlecase }}
+            </mat-header-cell>
+            <mat-cell class="min-width-3" *matCellDef="let vulnerability">{{ vulnerability.totalMedium }}
+            </mat-cell>
+          </ng-container>
+          <ng-container matColumnDef="totalLow">
+            <mat-header-cell class="min-width-xs-3"
+                             *matHeaderCellDef>{{ VulnerabilitySeverityAbbreviations.LOW |titlecase }}
+            </mat-header-cell>
+            <mat-cell class="min-width-3" *matCellDef="let vulnerability">{{ vulnerability.totalLow }}</mat-cell>
+          </ng-container>
+          <ng-container matColumnDef="totalNegligible">
+            <mat-header-cell class="min-width-xs-3"
+                             *matHeaderCellDef>{{ VulnerabilitySeverityAbbreviations.NEGLIGIBLE |titlecase }}
+            </mat-header-cell>
+            <mat-cell class="min-width-3" *matCellDef="let vulnerability">{{ vulnerability.totalNegligible }}
+            </mat-cell>
+          </ng-container>
+          <ng-container matColumnDef="totalFixableCritical" class="column-word-break">
+            <mat-header-cell class="min-width-xs-3"
+                             *matHeaderCellDef>{{ VulnerabilitySeverityAbbreviations.CRITICAL |titlecase }}Fixable
```

Comment:
Need a space between `{{ VulnerabilitySeverityAbbreviations.CRITICAL |titlecase }}` and `Fixable`

Reply:
@becky-intelletive space added. Thanks.